driverkit/config/4.0.0+driver/aarch64/debian_6.1.0-rpi7-rpi-2712_1.yaml

```diff
@@ -0,0 +1,9 @@
+kernelversion: "1"
+kernelrelease: 6.1.0-rpi7-rpi-2712
+target: debian
+architecture: arm64
+output:
+    module: output/4.0.0+driver/aarch64/falco_debian_6.1.0-rpi7-rpi-2712_1.ko
+    probe: output/4.0.0+driver/aarch64/falco_debian_6.1.0-rpi7-rpi-2712_1.o
+kernelurls:
+  - http://archive.raspberrypi.com/debian/pool/main/l/linux/linux-image-6.1.0-rpi7-rpi-2712_6.1.63-1+rpt1_arm64.deb
```

Comment:
We normally have 3 URLs for each kernel header; example:
```
kernelurls:
    - http://mirrors.edge.kernel.org/debian/pool/main/l/linux/linux-headers-6.5.0-0.deb12.4-amd64_6.5.10-1~bpo12+1_amd64.deb
    - http://mirrors.edge.kernel.org/debian/pool/main/l/linux/linux-headers-6.5.0-0.deb12.4-common_6.5.10-1~bpo12+1_all.deb
    - http://mirrors.edge.kernel.org/debian/pool/main/l/linux/linux-kbuild-6.5.0-0.deb12.4_6.5.10-1~bpo12+1_amd64.deb
```

We have a `kbuild`, a `common` and a flavor specific headers package.
In you case, these should be:
```
- http://archive.raspberrypi.com/debian/pool/main/l/linux/linux-headers-6.1.0-rpi7-common-rpi_6.1.63-1+rpt1_all.deb
- http://archive.raspberrypi.com/debian/pool/main/l/linux/linux-headers-6.1.0-rpi7-rpi-2712_6.1.63-1+rpt1_arm64.deb
- http://archive.raspberrypi.com/debian/pool/main/l/linux/linux-kbuild-6.1_6.1.63-1+rpt1_arm64.deb
```

Reply:
Ahh, guess I got confused by a file that only had the kbuild one in it and figured I'll just need the normal kernel image. Sorry!

Reply:
No problem! I am trying to fix that issue too!